src/index.js

```diff
@@ -83,4 +84,4 @@ class Orders extends Component {
   }
 }
 
-export default Orders;
+export default hot(module)(Orders);
```

Comment:
It doesn't look working to me, should I run local dev server differently than `yarn stripes serve stripes.config.js.local` ?

Reply:
no, it works with the same command